### DISCUSSION TITLE
GPIO Pin-Belegung

### DISCUSSION THREAD

**Discussion by fwenzelb** [MiczFlor/RPi-Jukebox-RFID#1430] [Q&A] [answered]
Hallo zusammen,
ich habe die Phoniebox auf einem Rasperry der ersten Generation installiert und daher weniger Pins als die neuen Raspberrys. Daher wollte ich die Belegung ändern. Hier ist die Pin-Belegung meines Pi:
![image](https://user-images.githubusercontent.com/85010872/120061840-5d9e1780-c05f-11eb-884c-eb28a4a2d61d.png)

Dann habe ich mich an diese Anleitung gehalten:
https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Using-GPIO-hardware-buttons
Darin steht folgendes:
Volume Down GPIO6 (PIN31) and GND
Volume Up GPIO5 (PIN29) and GND
Play/Pause/Halt (or how you call it) GPIO27 (PIN13) and GND
Next GPIO23 (PIN16) and GND
Previous GPIO22 (PIN15) and GND

Soweit so gut. Es funktioniert: 
- Play/Pause/
- Next
- Previous

Nun zum Problem: Auf der Anleitung von oben sehe ich auf meinem Raspberry keinen GPIO6 bzw PIN31 für "VolumeUp"
Wie gehe ich nun vor? Ich habe mich schon mal an die gpio_settings.ini (Pfad: /home/pi/RPi-Jukebox-RFID/settings) gewagt, aber das verwirrt mich noch mehr: Denn da steht:

[NextSong]
enabled: False
Type:  Button
Pin: 26
pull_up: True
hold_time: 0.3
functionCall: functionCallPlayerNext

Das sollte also gar nicht funktionieren, da hier Pin: 26 steht und ich es aber auf GPIO23 gelegt habe. Aber es funktioniert. 
Also ich bin total verwirrt. Kann mir jemand weiterhelfen?
Gruß Flo

**Comment by fwenzelb** [comment, marked as answer]
Hi, ich hab's geschafft! Ich hatte tatsächlich vergessen bei der Erstinstallation die GPIO-Controls zu installieren. Nachdem ich das System neu aufgesetzt habe, hat es funktioniert. Allerdings hat man beim Raspberry Pi der 1. Generation nur 5 x Ground, sodass ich nur 5 Knöpfe verbauen konnte (previous, pause, next, lauter, leiser)

**Reply by Sheepiely** [reply]
Can you daisychain the ground, so that you only need 1 ground for all the buttons?

**Reply by lenlennart** [reply]
Yes, that should be possible.
